plutus-core/plutus-core/src/PlutusCore/ParserCommon.hs

```diff
@@ -113,6 +113,7 @@ inBraces = between lbrace rbrace
 isIdentifierChar :: Char -> Bool
 isIdentifierChar c = isAlphaNum c || c == '_' || c == '\''
 
+-- | Return the source position of the input word.
```

Comment:
This would be fine as a partial comment, but if we're going to do Haddock I think we should say a bit more about what it's intended to do. "Given a word, creates a parser that matches precisely that word, returning its source position." or something

Reply:
Improved I think? Lmk if I should add more.